azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/ConfigMojo.java

```diff
@@ -477,16 +475,23 @@ private WebAppConfiguration getWebAppConfigurationWithoutValidation() throws Azu
 
     private WebAppConfiguration chooseExistingWebappForConfiguration()
             throws AzureExecutionException, AzureAuthFailureException {
-        final Azure az = getAzureClientByAuthType();
+        final Azure az = createAzureClient();
+        if (Objects.isNull(az)) {
```

Comment:
`createAzureClient` should throw exception if failed, right? so no need to do null check.